### PR TITLE
V2/global state for 3D sphere

### PR DESCRIPTION
**Pull request by janphilipp-winkler**
Please only consider components and changes described in the Dev Story

### CODE REVIEW

File: pages/[path].js [hunk 1]
```diff
@@ -12,38 +12,45 @@ import QuotesWrapper from "@/components/QuotesWrapper/QuotesWrapper";
 import SmileTrainerWrapper from "@/components/SmileTrainerWrapper/SmileTrainerWrapper";
 import GuidedMeditation from "@/components/GuidedMeditation/GuidedMeditation";
 import Entry from "@/components/SingleEntry/Entry";
+import { useDashboardState } from "@/components/DashboardStateProvider/DashboardStateProvider";
+import { useSphereState } from "@/components/ContextProviders/SphereStateProvider/SphereStateProvider";
 
 export default function Home() {
+  const { dashboardIsOpen, handleDashboardIsOpen } = useDashboardState();
+  const { sphereState, handleSphereState } = useSphereState();
```
Comment (MerleMak) [inline pages/[path].js:20]:
```suggestion
  const { sphereState } = useSphereState();
```
Maybe I overlooked it, but it seems like you don't use the handleSphereState here?

File: components/SingleEntry/Entry.js [hunk 2]
```diff
@@ -15,15 +15,21 @@ import { FaRegEye } from "react-icons/fa6";
 import { useState } from "react";
 import { AnimatePresence, motion } from "framer-motion";
 import { useSession } from "next-auth/react";
+import { useSphereState } from "../ContextProviders/SphereStateProvider/SphereStateProvider";
+import { useEffect } from "react";
 
 export default function Entry({ id }) {
+  const { sphereState, handleSphereState } = useSphereState();
```
Comment (MerleMak) [inline components/SingleEntry/Entry.js:22]:
```suggestion
  const { handleSphereState } = useSphereState();
```
same here but reverse, are you using the sphereState in this file?

**Review by MerleMak — COMMENTED** [review]
only some destructuring question, great job!!

File: components/SingleEntry/Entry.js (hunk 2 above)
Comment (janphilipp-winkler) [reply components/SingleEntry/Entry.js:22]: so true!

File: pages/[path].js (hunk 1 above)
Comment (janphilipp-winkler) [reply pages/[path].js:20]: I give it to <ActionBar>

**Pull request merged**